client/python/gradio_client/client.py

```diff
@@ -156,7 +156,7 @@ def __init__(
         )
         api_prefix: str = self.config.get("api_prefix", "")
         self.api_prefix = api_prefix.lstrip("/") + "/"
-        self.src_prefixed = urllib.parse.urljoin(self.src, api_prefix) + "/"
+        self.src_prefixed = urllib.parse.urljoin(self.src, api_prefix).rstrip("/") + "/"
```

Comment:
Why do we `rstrip` just to add back `/`?

Reply:
guarantees that the url ends in 1 and only 1 forward slash. previously was seeing some issues where sometimes there would be two forward slashes